web/usuaris/src/helpers/fetchers.ts

```diff
@@ -8,3 +8,41 @@ export const getUsers = async () => {
   const data = await response.json()
   return data
 }
+
+export const loginUserFetcher = async (user: object) => {
+  const errorMessage: { [key: number]: string } = {
+    401: 'Credenciales incorrectas.',
+    403: 'Usuario en proceso de activación. Por favor, póngase en contacto con el administrador.',
```

Comment:
These texts are translated?
Is the appropriate message if a non-admin user tries to log in?

Reply:
I will check the translation. The messages match with the errors available in swagger... Maybe I could precise more the 403 but I thought it was in the line of the message.